src/databricks/sql/client.py

```diff
@@ -737,6 +737,7 @@ def execute(
         self,
         operation: str,
         parameters: Optional[TParameterCollection] = None,
+        enforceEmbeddedSchema=False,
```

Comment:
can you rename it to `enforce_embedded_schema_correctness` to be consistent with the meaning of the thrift API. "enforce schema" has different with "enforce schema with correctness". And please use snake case which is python style.